scripts/gulpfiles/appengine_tasks.js

```diff
@@ -98,6 +136,15 @@ const deployDemos = gulp.series(
     deployAndClean
 );
 
+const deployDemosBeta = gulp.series(
+    prepareDeployDir,
+    copyStaticSrc,
+    copyAppengineSrc,
+    copyPlaygroundDeps,
```

Comment:
I'm going to file a bug to use a command line flag instead of having two separate gulp tasks.
I think it sounds like a good idea, but there isn't time to change it right now.

Reply:
https://github.com/google/blockly/issues/5155